galsim/interpolatedimage.py

```diff
@@ -25,6 +25,7 @@
 from . import _galsim
 from ._galsim import Interpolant
 from ._galsim import Nearest, Linear, Cubic, Quintic, Lanczos, SincInterpolant, Delta
+import numpy as np
```

Comment:
I know we do this in other files too, but I'd actually prefer to move in the other direction.  The more things we import at file scope the longer it takes to do `import galsim`.  Already it takes about a second.  So my preference would be to keep these imports inside the classes and functions that use them.